step-capstone/src/components/Trip.js

```diff
@@ -21,35 +21,45 @@ export default class Trip extends React.Component {
                 endDate: new Date(),
                 destinations: "",
                 description: ""
+            },
+            loaded: false,
+            selectedObject: null,
+            today: {
```

Comment:
maybe consider naming `today` as `displayDate` for consistency

Reply:
good suggestion! will do :)